docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -980,7 +980,82 @@ If you are using the `quarkus-hibernate-orm-panache` or `quarkus-mongodb-panache
 
 == Testing Security
 
-If you are using Quarkus Security, check out the link:security-testing[Testing Security] section for information on how to easily test security features of the application.
+Quarkus provides explicit support for testing with different users, and with the security subsystem disabled. To use
```

Comment:
How inlining all of this information into the `getting started` guide improves things ? User can't miss the existing mini sections linking to `security-testing.adoc`, it is in TOC, both in this doc and in `security.adoc`. This will also have the same problem as with inlining it into the `security.adoc` - the problem of overloading the `getting started` guide with more security specific test content.

Reply:
Because this is just as important as everything else that is covered in this document, and the fact that it is in its own document while everything else is here makes it easy to miss.

Really this should be a 'testing reference', the initial getting started guide shows you how to get started, this covers everything else.

Reply:
@stuartwdouglas Hi Stuart, when I type in Google `quarkus test security` the first link it offers is `Security Testing - Quarkus` linking to `security-testing.adoc` - this is a good search experience. This `security.adoc` root  to many leaf docs has worked well IMHO as `secuiry.adoc` has been overloaded before.

`Getting Started Testing` guide is also linked to (as the 4th link). So your main concern is that the users starting from `Getting Started Testing` may miss the security testing doc ? But the reference to the Test Security mini section is in TOC. What about testing the databases, vault, etc, etc, I'm not sure it all should go inlined into `Getting Started` - and the security is no different in this regard. 
Would adding a main Quarkus page badge linking to the security doc, as suggested below, make it more visible ?

Reply:
@stuartwdouglas Well, I was not correct when saying it was all inlined in `Getting Started Testing` since your PR links to OIDC/JWT test sections.

I like the idea of having a standalone `security-testing.adoc`, but I don't want to make the blocking issue from it being removed. Please update the links in `security.adoc` mini Test Security section, `security-jwt.adoc`, `security-openid-connect.adoc`, `security-openid-connect-web-authentication.adoc` test sections to point to `Getting Started testing` subsection introducing `TestSecurity`